cmd/ringbuf/src/lib.rs

```diff
@@ -150,11 +147,14 @@ fn ringbuf(
     ringbufs.sort();
 
     if subargs.list {
-        info!("{:18} {:<30} {:<10} {}", "MODULE", "BUFFER", "ADDR", "SIZE");
+        #[rustfmt::skip]
```

Comment:
this line and 156 both annoy me, however, i could not get `#[rustfmt::skip::macros]` to work like we do with `println` elsewhere, so resorted to this. These are *just* over 80 chars and so get splatted into five lines without it, which I figured you would not like, but if you don't mind, could remove these two.